docker-compose.qemu.yml

```diff
@@ -2,7 +2,7 @@ version: "2"
 
 services:
   worker:
-    image: bh.cr/balena/leviathan-worker-${BALENA_ARCH:-amd64}/2.9.50
+    image: bh.cr/balena/leviathan-worker-${BALENA_ARCH:-amd64}/${WORKER_REF:-2.9.50}
```

Comment:
Unfortunately this will break the [Renovate parsing of this line](https://github.com/balena-os/leviathan/blob/90d16858e6e622a1f95a86e578782ee931abb47f/.github/renovate.json#L9-L11).

The proper fix is to probably define WORKER_REF as build ARG in a Dockerfile, then configure renovate to bump the default build arg value using existing version matching presets.

I can push a commit.